build-system/tasks/extension-helpers.js

```diff
@@ -464,18 +466,12 @@ async function buildExtension(
   }
   const extDir = 'extensions/' + name + '/' + version;
 
-  // Use a separate watcher for extensions to copy / inline CSS and compile JS
-  // instead of relying on the watchers used by compileUnminifiedJs and
-  // compileMinifiedJs, which only recompile JS.
+  // Use a separate watcher for css and jison compilation.
+  // The watcher within compileJs recompiles the JS.
   if (options.watch) {
-    options.watch = false;
-    watchExtension(extDir, name, version, latestVersion, hasCss, options);
-    // When an ad network extension is being watched, also watch amp-a4a.
-    if (name.match(/amp-ad-network-.*-impl/)) {
-      const a4aPath = `extensions/amp-a4a/${version}`;
-      watchExtension(a4aPath, name, version, latestVersion, hasCss, options);
-    }
```

Comment:
Whoa! I assume this now happens automatically? (Very cool!)

Reply:
Yep!